static/index.html

```diff
@@ -0,0 +1,184 @@
+<html>
+<head>
+    <meta http-equiv="content-type" content="text/html; charset=UTF-8"/>
+    <title>Sorry, the enquiry form is undergoing maintenance – GOV.UK</title>
```

Comment:
Should this be 'sorry, there is something wrong...' like the title in the body? Seems like mixed messages